packages/authentication/src/definitions.ts

```diff
@@ -1409,6 +1420,13 @@ export interface AdditionalUserInfo {
  */
 export type AuthStateChangeListener = (change: AuthStateChange) => void;
 
+/**
+ * Callback to receive the id token change notifications.
+ *
+ * @since 6.2.0
```

Comment:
```suggestion
   * @since 6.3.0
```

The next release will be 6.3.0, see https://github.com/capawesome-team/capacitor-firebase/pull/743.